addon/components/service-worker-update-notify.js

```diff
@@ -27,45 +32,51 @@ export default Component.extend({
   // public
   pollingInterval: 1200000, // 20 minutes in ms
 
-  didInsertElement() {
+  config: computed(function() {
```

Comment:
Right! That *should* fix the tests I believe. Good catch!

Reply:
I'm a little sad the native getter isn't supported in <= ember 3.8 :(

Reply:
cause this is like.. forced memoization